src/me/mrCookieSlime/Slimefun/listeners/ItemListener.java

```diff
@@ -283,8 +285,18 @@ else if (SlimefunManager.isItemSimiliar(item, SlimefunItems.MEDICINE, true)) {
 				else if (item.getType() == Material.POTION) {
 					SlimefunItem sfItem = SlimefunItem.getByItem(item);
 					if (sfItem != null && sfItem instanceof Juice) {
+						// Fix for 1.11 and 1.12 where Saturation potions are no longer working
+						if (ReflectionUtils.getVersion().startsWith("v1_11_") || ReflectionUtils.getVersion().startsWith("v1_12_")) {
```

Comment:
Heavy call.

Should be saved in a class-level variable as it is an expensive operation, since it performs code reflection.
Also, to make future support easier, I'd instead check for 1_10 and 1_9.

Reply:
For heavy calls, here is a better fix : https://github.com/TheBusyBiscuit/CS-CoreLib/pull/32/files

I prefered to check for 1.11 and 1.12 because I've seen on MC's bug tracker that a fix is planned for 1.13

Reply:
My mistake, I just doubled check https://bugs.mojang.com/browse/MC-112389.
No fix is currently planned.

Let's switch to 1.9 and 1.10 to make 1.13 support easier to do.